docs/integration/glossary.md

```diff
@@ -0,0 +1,49 @@
+# DTS Integration Glossary
+
+We use the following terms in the DTS Integration Guide.
+
+* **DTS**: The [BER data transfer system](https://kbase.github.io/dts/)
```

Comment:
I think just call it the Data Transfer System for now -- when ~~DTS~~ the DTS takes over the world, you don't want to have to go back through these docs and find all the places where you mentioned BER.